test/explorer/series_test.exs

```diff
@@ -30,6 +30,20 @@ defmodule Explorer.SeriesTest do
   end
 
   describe "from_list/1" do
+    test "with nils" do
+      s = Series.from_list([nil, nil, nil])
+
+      assert Series.to_list(s) === [nil, nil, nil]
+      assert Series.dtype(s) == :null
+    end
```

Comment:
What about a list of lists of nils?

    Series.from_list([[nil, nil], [nil, nil, nil]])

Reply:
test/explorer/series/list_test.exs has below test case: 

```
test "list of list of nulls" do
       series = Series.from_list([[nil, nil], [nil]])
       assert series.dtype == {:list, :null}
       assert series[0] == [nil, nil]
       assert Series.to_list(series) == [[nil, nil], [nil]]
     end
```